llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -5198,6 +5296,7 @@ static void ReplaceLoadVector(SDNode *N, SelectionDAG &DAG,
   case MVT::v4f16:
   case MVT::v4f32:
   case MVT::v8f16: // <4 x f16x2>
+  case MVT::v8i16: // <4 x i16x2>
```

Comment:
Looks like we're missing bf16 here.

Reply:
Added it.

Reply:
I do not see it in the diff.

Reply:
It's there now, I believe there was some lag in GitHub UI

Reply:
Yup. I see the changes now.